pkg/controllers/user/authz/podsecuritypolicy/template.go

```diff
@@ -113,28 +114,28 @@ func doesPolicyExist(policyLister v1beta12.PodSecurityPolicyLister, name string)
 	return !errors.IsNotFound(err)
 }
 
-func getPodSecurityPolicyTemplateID(projectLister v3.ProjectLister, clusterLister v3.ClusterLister, projectID string,
+func getPodSecurityPolicyTemplateID(psptcbLister v3.PodSecurityPolicyTemplateProjectBindingLister, clusterLister v3.ClusterLister, projectID string,
```

Comment:
I feel that this paradigm of passing a lister in kind of obfuscates where and how the manager struct's fields are getting used. Id rather see
```
func (m *manager)  getPodSecurityPolicyTemplateID(projectID, clusterName string) ...
...
m.clusterList()
```